smoke.json

```diff
@@ -1,3 +1,3 @@
 {
-  "procfile" :  "docker://gcr.io/paketo-buildpacks/procfile"
+  "procfile" :  "docker://anthonydahanne/procfile:dual-march15"
```

Comment:
This will be reverted to the official Procfile buildpack when an official multi-arch version is available @anthonydahanne